tests/jeos/firstrun.pm

```diff
@@ -167,7 +167,7 @@ sub run {
     send_key 'ret';
 
     # Accept EULA if required
-    unless (is_tumbleweed || is_microos || is_leap('>=15.4')) {
+    unless (is_tumbleweed || is_slowroll || is_microos || is_leap('>=15.4')) {
         assert_screen 'jeos-doyouaccept';
         send_key 'ret';
     }
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check.

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

```suggestion
    if (is_leap('<15.4')) {
        assert_screen 'jeos-doyouaccept';
        send_key 'ret';
    }
```

and as Leap < 15.4 is EOL just delete the complete if-block